packages/vant/src/stepper/Stepper.tsx

```diff
@@ -93,7 +94,7 @@ export default defineComponent({
       value = formatNumber(String(value), !props.integer);
       value = value === '' ? 0 : +value;
       value = Number.isNaN(value) ? +min : value;
-      value = Math.max(Math.min(+max, value), +min);
+      value = props.autoFixed ? Math.max(Math.min(+max, value), +min) : value;
```

Comment:
According the original issue, this prop should only affect the value entered by the user, and should not affect the value after clicking the plus and minus buttons.

But the condition here may affect the buttons.

Reply:
Ah.... you mean that the value is not fixed only when onBlur, but still fixed when the plus or minus button is clicked?

Reply:
If so, I think that since there is no more automatic correction, the correction is not needed in any case, and the final result is handled by the user. 😂

Reply:
The main purpose is to not format the value entered by the user.

For values generated by the interaction of the component itself, it should be formatted.